src/js/pensions/config/form.js

```diff
@@ -133,9 +138,48 @@ const formConfig = {
               combatSince911
             }
           }
+        },
+        reserveAndNationalGuard: {
+          path: 'military/reserveAndNationalGuard',
+          title: 'Reserve and National Guard',
+          uiSchema: {
+            'ui:title': 'Reserve and National Guard',
+            nationalGuardActivation: {
+              'ui:title': 'Are you currently on federal active duty in the National Guard?',
+              'ui:widget': 'yesNo'
+            },
+            nationalGuard: {
+              'ui:title': 'What is the information for your unit?',
+              'ui:options': {
+                expandUnder: 'nationalGuardActivation',
+              },
+              name: {
+                'ui:title': 'Name of Reserve/NG unit'
+              },
+              address: address.uiSchema(),
+              phone: phoneUI(),
+              date: dateUI
+            }
+          },
+          schema: {
+            type: 'object',
+            required: ['nationalGuardActivation'],
+            properties: {
+              nationalGuardActivation,
+              nationalGuard: {
+                type: 'object',
+                required: ['name', 'address', 'phone', 'date'],
```

Comment:
Like I mentioned in Slack, I think you'll want to use `ui:required` functions in the schema to mark these fields as required if `nationalGuardActivation` is true. For the address part, the address uiSchema should have a parameter for that.